whipper/command/offset.py

```diff
@@ -161,17 +146,18 @@ def match(archecksum, track, responses):
 
                 # now try and rip all other tracks as well, except for the
                 # last one (to avoid readers that can't do overread
-                for track in range(2, (len(table.tracks) + 1) - 1):
```

Comment:
this should be reverted back to `for track in range(2, (len(table.tracks) + 1) - 1):`